.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+{Description of the PR}
+
+Issues this may resolve (delete line if none):
+
+TODO:
+- [ ] Perform basic functions test on production robot
```

Comment:
Who is checking the boxes? I would propose the reviewer either does the test themselves (ideal) or watches the developer do the test.